roles/acm_hypershift/README.md

```diff
@@ -27,45 +29,127 @@ api.<ah_cluster_name>.<ah_base_domain>
-| ah_cluster_type         | kubevirt                                  | No        | The type infrastructure provider, currently only only supports "kubevirt"               |
-| ah_force_deploy         | false                                     | No        | Force redeploy of a cluster                                                             |
-| ah_no_log               | true                                      | No        | Allow logging on sensitive tasks                                                        |
-| ah_ocp_version          | 4.14.0                                    | No        | Full OCP version to install on the hypershift cluster. <major>.<minor>.<patch>          |
-| ah_pullsecret_file      | ""                                        | Yes       | Required to pull the hosted cluster release image                                       |
-| ah_release_image        | quay.io/openshift-release-dev/ocp-release | No        | The release image to install, from the specified OCP version                            |
-| ah_node_memory          | 8Gi                                       | No        | Memory which is visible inside the Guest OS (type BinarySI, e.g. 5Gi, 100Mi)            |
+| Variable                | Default                                   | Required  | Description
+| ----------------------- | ----------------------------------------- | --------- | ------------
+| ah_cluster_name         | Random string                             | No        | Name of the hosted cluster
+| ah_base_domain          |                                           | No        | The base domain, DNS resolution for OCP endpoints must be prepared in advance. If undefined, the hosted cluster routes will be created as subdomains for \<hc\>.apps.\<mgmt-cluster-domain\>
+| ah_node_pool_replicas   | 2                                         | No        | Number of pool replicas, minimum 2 is required
+| ah_cluster_network_cidr | 10.132.0.0/14                             | No        | Cluster network CIDR
+| ah_clusters_ns          | clusters                                  | No        | The prefix for the namespace
+| ah_cluster_type         | kubevirt                                  | No        | The type infrastructure provider, currently supports "kubevirt" and "agent" (baremetal)
+| ah_force_deploy         | false                                     | No        | Force redeploy of a cluster, if a HCP already exist with the same name it will be deleted
+| ah_no_log               | true                                      | No        | Allow logging on sensitive tasks
+| ah_pullsecret_file      | Pullsecret from management cluster        | No        | Pull secret file
+| ah_release_image        | OCP release from Management cluster       | No        | The release image to install. In disconnected, it must be the image already mirrored.
```

Comment:
I've seen these two `ah_pullsecret_file` and `ah_release_image` is not defined anymore in the defaults file, it's created on the fly if it's undefined, but there's no example regarding what value should be used in case it's defined. Is this expected?